cpp/net/connection_pool.cc

```diff
@@ -188,7 +188,7 @@ ConnectionPool::ConnectionPool(libevent::Base* base)
 
 // static
 evhtp_res ConnectionPool::Connection::ConnectionErrorHook(
-    evhtp_connection_t* conn, evhtp_error_flags errtype, void* arg) {
+    evhtp_connection_t* conn, evhtp_error_flags, void* arg) {
```

Comment:
This looks like exactly the kind of information one would like to know when debugging... :smiley: